app/Actions/Like/LikePlaceAction.php

```diff
@@ -51,7 +52,23 @@ public function execute(LikePlaceRequest $request): LikePlaceResponse
         if (!$place) {
             throw new PlaceNotFoundException;
         }
+
+        $liked = LikeStatus::none();
+        $userId = Auth::id();
+        $like = $this->likeRepository->findByUserAndPlace($userId, $request->getPlaceId());
```

Comment:
Same already exists above
` $like = $this->likeRepository->findByUserAndPlace(Auth::id(), $request->getPlaceId());`

Reply:
this like is after update...